docs/pros-and-cons.md

```diff
@@ -3,20 +3,20 @@ Pros and cons
 
 **Reasons to use Insect**
 
-  - Insect is **open source**.
+  - Insect is **open-source**.
   - There is a [web version](https://Insect.sh/) that requires **no installation**.
-  - With both browser and terminal versions available, insect is truly **cross-platform**.
+  - With both browser and terminal versions available, Insect is truly **cross-platform**.
   - Insect has *first-class* support for **physical units**, including metric and binary prefixes.
     While evaluating your calculation, Insect ensures that you did not accidentally make any
     mistakes in combining the physical quantities.
   - Insect supports an [**interactive**](https://en.wikipedia.org/wiki/REPL) style with its
-    readline-based interface. There is a saved history that can be browsed by pressing up- and
-    down keys. The history is also searchable via *Ctrl-R*.
-  - The syntax of Insect is rather strict. The parser does not try to be "smart" on syntactically
+    readline-based interface. There is a saved history that can be browsed by pressing the up and
+    down arrow keys. The history is also searchable via *Ctrl-R*.
+  - Insect's syntax is rather strict. The parser does not try to be "smart" on syntactically
```

Comment:
Should this be
```suggestion
  - Insects syntax is rather strict. The parser does not try to be "smart" on syntactically
```
(?)

Reply:
Hmm, I'm pretty sure the version with the apostrophe is the correct one.

Reply:
Oh, this is different in German. At least it used to be. I just checked and a spelling reform actually allows the "English" apostrophe in certain special cases. But we would not put an apostrophe in that case: "Insects Grammatik". Anyway... learned something :-).